tools/sandbox_test/test_programmer/sao047b/LICENSE

```diff
@@ -0,0 +1,11 @@
+
+
+
+Blit16
+
+ISC License Copyright (c) 2018 Andrew Reece
```

Comment:
Who's Andrew Reece?

Reply:
Oh, I can get rid of that. It was a font I was playing with.